src/main/java/openperipheral/integration/forestry/AdapterBeeHousing.java

```diff
@@ -61,24 +61,26 @@ public IIndividual getQueen(IBeeHousing beeHousing) {
 
 	@Asynchronous
 	@ScriptCallable(returnTypes = ReturnType.TABLE, description = "Get the full breeding list thingy. Experimental!")
-	public List<Map<String, Object>> getBeeBreedingData(IBeeHousing housing) {
+	public List<Map<String, Object>> getBeeBreedingData(IBeeHousing housing, @Optionals @Arg(name = "showSecrets", type = ArgType.BOOLEAN) Boolean showSecrets) {
```

Comment:
You mean you intend to make making mutations secret useless? As you said yourself, even NEIAddons has that disabled by default. In the config, which is a big difference to just a parameter.